### PR TITLE
Create a script for moving dist static files to the static folder.

### PR DESCRIPTION
**Pull request by StaticallyTypedRice**
When the frontend is built manually or via `install.sh`, it does not put static files in the `dist/static` folder, which the index.html file refers to. This to leads the frontend refusing to run in some cases unless you manually move the static files into the right folder. I made a quick script to automate this.

Feel free to delete the changes to main.tsx, that's for another contribution I'm working on.

### CODE REVIEW

File: ui/src/components/main.tsx [hunk 1]
```diff
@@ -102,6 +103,7 @@ export class Main extends Component<any, MainState> {
     dataType: getDataTypeFromProps(this.props),
     sort: getSortTypeFromProps(this.props),
     page: getPageFromProps(this.props),
+    hasUnsavedData: false,
```
Comment (Nutomic) [inline ui/src/components/main.tsx]: Its better if you clena this up yourself. Just delete the lines, make a new commit and push it to the same branch.

**Comment by dessalines** [comment]
It does appear to be an issue, but this isn't the way to handle it. For some reason the newer versions of yarn aren't working correctly with the `fuse.js`, and not setting the production flag.

`fuse.js` IE fusebox is the front end build system, which should probably be upgraded and fixed at some point.

**Comment by StaticallyTypedRice** [comment]
> edit: actually nevermind, its the webserver that serves everything from /static, not the front end. Its working fine as is.
> 
> If you run `./install.sh`, then goto http://localhost:8536, it should work fine.

However, when you try to run the frontend by itself, it doesn't work.

**Comment by dessalines** [comment]
What is "running the front end by itself?"

You should use yarn start and goto localhost:4444 for development.

File: ui/src/components/main.tsx (hunk 1 above)
Comment (StaticallyTypedRice) [reply ui/src/components/main.tsx]: Fixed

**Comment by StaticallyTypedRice** [comment]
> What is "running the front end by itself?"
> 
> You should use yarn start and goto localhost:4444 for development.

I just started the `http-server` node module with the index at `ui/dist`.

**Comment by dessalines** [comment]
I'm closing this, since its not using the right front end way to do live coding through the front-end build system, which is [already in the docs](https://dev.lemmy.ml/docs/contributing_local_development.html#running)

Use `yarn start` for front end coding.

**Pull request closed without merging**